hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneClientAdapterImpl.java

```diff
@@ -782,6 +791,32 @@ private List<FileStatusAdapter> listStatusVolume(String volumeStr,
     return res;
   }
 
+  /**
+   * Helper for OFS listStatus
```

Comment:
```suggestion
   * Helper for OFS listStatus on a bucket to get all snapshots.
 ```
 
 Could be a one line.